packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -298,9 +305,24 @@ export default class QuanticCaseClassification extends LightningElement {
    * @returns {Array<object>}
    */
   get suggestions() {
-    return this.classifications.map((suggestion) => {
-      return {...suggestion, checked: suggestion.value === this._value};
-    });
+    if (!this.allOptionsReceived) {
+      return [];
+    }
+    return this.classifications
+      .filter((suggestion) => {
+        const suggestionIncludedInOptions = this.options.some(
+          (option) => option.value === suggestion.value
+        );
+        if (!suggestionIncludedInOptions) {
+          this.logWarningOnce(
+            `The value "${suggestion.value}" was not found among all the options retrieved from Salesforce. Please confirm that the coveoFieldName corresponds to the correct sfFieldApiName.`
```

Comment:
```suggestion
            `The value "${suggestion.value}" was not found. Ensure that the Coveo field name (`coveoFieldName`) corresponds to the correct Salesforce field name (`sfFieldApiName`).`
```
Is the "among all options retrieved from Salesforce" necessary?

Reply:
Thank you! Yes the "among all options retrieved from Salesforce" is necessary, it's  the main hint to help the user understand the error.